src/main/java/org/jabref/gui/util/IconValidationDecorator.java

```diff
@@ -31,19 +31,11 @@ public IconValidationDecorator(Pos position) {
         this.position = position;
     }
 
-    @Override
```

Comment:
Is the super method also gone?

Reply:
the super method `getGraphicBySeverity` is a protected method with a different behavior, so I removed it.

Reply:
The right answer would have been: The super method is from JavaFX, not from JabRef.